### DISCUSSION TITLE
Hide microphone and speaker tray icons

### DISCUSSION THREAD

**Discussion by ygoe** [ramensoftware/windhawk-mods#964] [Mod ideas]
Would it be possible to hide both the microphone and speaker icons in the taskbar? I've installed VoiceMeeter to redirect and mix audio streams across devices, and it interferes massively with Windows' sound devices. Basically, the microphone icon is now always on because the microphone is constantly active and forwarded to another PC (because my PC doesn't know if an app on the other computer wants to use it) and the speaker and volume control is entirely in VoiceMeeter and Windows settings have no effect. So both icons are useless now.

**Comment by m417z** [comment]
I wanted to create a dedicated mod for these, but for now, you can try the following:
* [Hide the volume icon](https://github.com/ramensoftware/windows-11-taskbar-styling-guide?tab=readme-ov-file#hide-the-volume-icon)
* [Hide the microphone icon](https://github.com/valinet/ExplorerPatcher/discussions/1867#discussioncomment-7967498)
